core/src/main/java/io/fabric8/maven/core/handler/ContainerHandler.java

```diff
@@ -94,14 +87,19 @@ private String getImagePullPolicy(ResourceConfig config) {
     }
 
     private String getImageName(ImageConfiguration imageConfiguration) {
-        if (Strings.isNullOrBlank(imageConfiguration.getName())) {
-            return imageConfiguration.getName();
-        }
+        if (Strings.isNullOrBlank(imageConfiguration.getName()))
+            return null;
```

Comment:
please added braces also for single line ifs.

Reply:
done